stable/datadog/values.yaml

```diff
@@ -361,6 +361,16 @@ datadog:
     #
     apparmor: unconfined
 
+    ## @param enableTCPQueueLength - boolean - optional
+    ## Enable the TCP queue length eBPF-based check
+    #
+    enableTCPQueueLength: true
```

Comment:
should we comment these out by default?

Reply:
Indeed, the default value should be `false` and not `true`.
But it’s cleaner to explicitly set the the default value in `values.yaml` rather than commenting out the value and add extra-logic in the templates where the parameter is used to handle the case where it isn’t defined.